drivers/soundwire/intel.c

```diff
@@ -1382,10 +1383,10 @@ static int intel_master_probe(struct platform_device *pdev)
 	return 0;
 }
 
-int intel_master_startup(struct platform_device *pdev)
+int intel_link_startup(struct intel_link_dev *ldev)
```

Comment:
why not keep struct ancillary_device *ancildev as the parameter to keep consistency with the platform device implementation?

Reply:
done`